src/System.Text.Json/tests/Serialization/OptionsTests.cs

```diff
@@ -254,12 +254,12 @@ public static IEnumerable<object[]> JsonEncodedTextStringsCustom
                 return new List<object[]>
                 {
                     new object[] { "age", "\\u0061\\u0067\\u0065" },
-                    new object[] { "éééééêêêêê", "éééééêêêêê" },
-                    new object[] { "ééééé\"êêêêê", "ééééé\\u0022êêêêê" },
-                    new object[] { "ééééé\\u0022êêêêê", "ééééé\\\\\\u0075\\u0030\\u0030\\u0032\\u0032êêêêê" },
-                    new object[] { "ééééé>>>>>êêêêê", "ééééé\\u003E\\u003E\\u003E\\u003E\\u003Eêêêêê" },
-                    new object[] { "ééééé\\u003e\\u003eêêêêê", "ééééé\\\\\\u0075\\u0030\\u0030\\u0033\\u0065\\\\\\u0075\\u0030\\u0030\\u0033\\u0065êêêêê" },
-                    new object[] { "ééééé\\u003E\\u003Eêêêêê", "ééééé\\\\\\u0075\\u0030\\u0030\\u0033\\u0045\\\\\\u0075\\u0030\\u0030\\u0033\\u0045êêêêê" },
+                    new object[] { "\u00E9\u00E9\u00E9\u00E9\u00E9\u00EA\u00EA\u00EA\u00EA\u00EA", "\u00E9\u00E9\u00E9\u00E9\u00E9\u00EA\u00EA\u00EA\u00EA\u00EA" },
+                    new object[] { "\u00E9\u00E9\u00E9\u00E9\u00E9\"\u00EA\u00EA\u00EA\u00EA\u00EA", "\u00E9\u00E9\u00E9\u00E9\u00E9\\u0022\u00EA\u00EA\u00EA\u00EA\u00EA" },
+                    new object[] { "\u00E9\u00E9\u00E9\u00E9\u00E9\\u0022\u00EA\u00EA\u00EA\u00EA\u00EA", "\u00E9\u00E9\u00E9\u00E9\u00E9\\\\\\u0075\\u0030\\u0030\\u0032\\u0032\u00EA\u00EA\u00EA\u00EA\u00EA" },
```

Comment:
This type of change does end up making harder to parse/reason about what the expected value should be (`\u` vs `\\u`):
https://github.com/dotnet/corefx/pull/39524#discussion_r303964442

Is there a cleaner way to format the non-ascii text to avoid this drawback of the guideline?

Reply:
> Is there a cleaner way to format the non-ascii text to avoid this drawback of the guideline?

I'm not aware of one. The guideline exists because it's very easy to accidentally lose / mistransform non-ASCII characters when files roundtrip through various editors and text tools.

Reply:
@ahsonkhan HtmlDecode perhaps will do better in this test